### ISSUE TITLE
Add Functional Language selection menu at start

### ISSUE THREAD

**Issue by Qiangong2** [Tuxemon/Tuxemon#377]
I propose a language select menu be added on the boot-up of the game. It would include it inside the currently non-functional "options" menu.

**Comment by airon90** [comment]
Automatically the software should recognize the system language and to use it by default. Fallback to English should be implemented, if not already implemented.

**Comment by bitcraft** [comment]
Yes, I think it was discussed before that locale should be inferred from the system.  Having the ability to change languages would be nice for people who can read many languages.
